src/main/kotlin/uk/gov/dluhc/printapi/database/repository/CertificateRepository.kt

```diff
@@ -23,6 +24,8 @@ interface CertificateRepository : JpaRepository<Certificate, UUID> {
 
     fun findByGssCodeInAndSourceTypeAndSourceReference(gssCodes: List<String>, sourceType: SourceType, sourceReference: String): Certificate?
 
+    fun findBySourceTypeAndInitialRetentionRemovalDateBefore(sourceType: SourceType, initialRetentionRemovalDate: LocalDate): List<Certificate>
```

Comment:
Q: Do we need any index around this? However index with date might be complex?

Reply:
I added `certificate_initial_retention_removal_date_idx` in a previous PR 👍